spring-cloud-gcp-core/pom.xml

```diff
@@ -14,12 +14,12 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>commons-logging</groupId>
-			<artifactId>commons-logging</artifactId>
-		</dependency>
-		<dependency>
 			<groupId>com.google.cloud</groupId>
 			<artifactId>google-cloud</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
```

Comment:
Good catch! This was because of NestedRuntimeException, but since we got rid of it, I guess I can also delete this.

Actually, I removed commons-logging because the core module doesn't use it. I never got to use SF 5.0 here, only on the pubsub starter.